server/cluster/cluster.go

```diff
@@ -542,7 +547,43 @@ func (c *Cluster) pickOneShardOnNode(nodeName string) (uint32, error) {
 	return 0, ErrNodeNotFound.WithCausef("nodeName:%s", nodeName)
 }
 
-type metaData struct {
-	cluster         *clusterpb.Cluster
-	clusterTopology *clusterpb.ClusterTopology
+func (c *Cluster) RouteTables(_ context.Context, schemaName string, tableNames []string) (map[string]*RouteEntry, uint64, error) {
```

Comment:
A named struct called `RouteTablesResult` which contains `[]RouteEntry` and `TopologyVersion` is better than the anonymous two variables.